sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/Analyzer.scala

```diff
@@ -1890,8 +1890,9 @@ class Analyzer(
   object ResolveFunctions extends Rule[LogicalPlan] {
     val trimWarningEnabled = new AtomicBoolean(true)
     def apply(plan: LogicalPlan): LogicalPlan = plan.resolveOperatorsUp {
-      case UnresolvedFunc(CatalogAndFunctionIdentifier(catalog, identifier)) =>
-        ResolvedFunc(catalog, identifier)
+      case RefreshFunction(UnresolvedFunc(multipartIdent)) =>
```

Comment:
I tried this way but thought it may not needed to list the command to analyze `UnresolvedFun`.

It' ok to follow this.